datafeeder/src/main/java/org/georchestra/datafeeder/service/DatasetsService.java

```diff
@@ -445,6 +459,10 @@ private boolean isShapefile(@NonNull Path path) {
         return path.getFileName().toString().toLowerCase().endsWith(".shp");
     }
 
+    private boolean isCsv(@NonNull Path path) {
+        return path.getFileName().toString().toLowerCase().endsWith(".csv");
```

Comment:
there is a `FilenameUtils.getExtension(String path)` in apache commons-io which seems to handle specific corner-cases of (Windows) filenames, in case the current approach is judged too fragile (this is just a suggestion, I don't mind keeping it as it is, it seems that it is already what was done with shapefiles anyway).